docs/Contribute/ContributeWrite/WriteStack.md

```diff
@@ -0,0 +1,63 @@
+# Contribute to the Software Stack
+
+:::warning
+Writing on the *software stack* makes changes directly to the BTCPay Server software that everyone uses. **Do not** change the code if you aren't familiar with it.
+:::
+
+::: tip
+If you have trouble finding a string or contributing to the software stack, ask the [community](../../Community.md)
+:::
+
+## **Step 1**
+
+Fork/Clone the main repository ([BTCPay Server](https://github.com/btcpayserver/btcpayserver/)) using Github Desktop and publish it.
```

Comment:
> using Github Desktop

Actually GitHub desktop is just one way. There's GitHub web which for beginners may be even easier for newbies. So maybe just mention GitHub web version or Desktop or just keep "GitHub" as generic term.

Reply:
```suggestion
Fork/Clone the main repository ([BTCPay Server](https://github.com/btcpayserver/btcpayserver/)) using Github and publish it.
```